src/app.json.template

```diff
@@ -28,5 +28,6 @@
         "linkedin": {
             "clientId": "[ CLIENTID HERE ]"
         }
-    }
+    },
+    "defaultGridState": "grid-view"
```

Comment:
You are changing the implicit and expicit default from list-view and grid-view... and any current user of Career Portal will get the new view when upgrading. Intended?